tests/security/oscap_pci_dss_4/oscap_xccdf_remediate.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'pci-dss-4' hardening in the 'scap-security-guide': mitigation mode
+# Maintainer: QE Security <none@suse.de>
+# Tags: poo#93886, poo#104943
+
+use base 'oscap_tests';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+
```

Comment:
neatpicking: avoid more than one blank like to separate code blocks.

Reply:
File will be deleted during refactoring.